docker/compose/zrok-public-reserved/compose.yml

```diff
@@ -38,6 +38,7 @@ services:
       STATE_DIRECTORY: /mnt  # zrok homedir in container
 
       # most relevant options
+      ZROK_SUBDOMAIN:       # frontend subdomain, e.g. "myapp" for "myapp.in.zrok.io"
```

Comment:
Can we standardize on `UNIQUE_NAME`? It's not really a subdomain, at all.

Reply:
True, it's a share name, not a subdomain. When I'm imagining the UX, and think of where the end-user will encounter the share name, I mainly think of the frontend endpoint URL. From that I'm leaping to the conclusion that the most user-friendly name for this value is like "subdomain" or "hostname." 

We could use a variable name that reflects the internal model like `UNIQUE_NAME` as long as we explain its use. frontdoor users won't see the CLI usage hint with the same frequency as CLI users due to the fact of the wrapper. I felt the need to invent a meaningful name to reduce the need to explain what it's for, mainly for frontdoor users.

For CLI users it would be helpful to add something about this name becoming part of the frontend endpoint URL. We could say "A unique subdomain for the reserved share."
> A unique name for the reserved share (defaults to generated identifier)

Reply:
It's currently `--unique-name` in `zrok reserve` and I'd like to keep it that way for now. It's a unique name for the share. I don't want to communicate to the user that it's a subdomain at all... it's not. It's a unique name for the share token. It would be ideal if we just reflect that naming throughout.

Reply:
OK, I must've misunderstood how this value is used, so I'll wait to suggest a change until I understand. I believed incorrectly that `--unique-name` was used to define a reserved share name that is rendered in the frontend endpoint template like `https://{token}.frontends.zrok.example.com`.

Reply:
It is! That's not a subdomain.. it's a hostname that corresponds to a share token. `{token}` is the share token... which when you specify one, rather than letting it be generated, is called a "unique name". So, I'd just like to keep that naming used throughout.

Reply:
We can call it UNIQUE_NAME, but I can't think of a better way to explain to the user what it's for than 

> frontend subdomain, e.g. "myapp" for "myapp.in.zrok.io"

We could go with "frontend hostname" or "reserved hostname" if you prefer, but "unique name" demands explanation. A meaningful name would be better.

Reply:
It _is_ a meaningful name... it's a unique name for the share token. Public shares can also be accessed privately, and you'd use `zrok access private uniqueName`. It's not a hostname or a subdomain... it's a unique name for the share.

Reply:
How about this explanation?

> vanity share token is used to construct frontend subdomain, e.g. "myapp" for "myapp.in.zrok.io"

Reply:
I don't think we should call it a "vanity share token" either. It's a unique name for the share.